previewer/presentationvideo.js

```diff
@@ -47,16 +54,67 @@
         });
     });
 
-    // when both the plugins and the JSON-LD are loaded, we can show the previewer
-    $.when(extractor_req, videojs_thumb_plugin).done(function(extract_data, videojs_plugin){
+    // load slick to handle navigation
+    // First the stylesheets
+    s = document.createElement("link");
+    s.rel = "stylesheet";
+    s.type = "text/css";
+    s.href = Configuration.previewer + "/slick/slick.css";
```

Comment:
maybe clean this a bit up? Put all those css files in a list and loop over it for loading?

Reply:
Ok